src/main/java/hudson/plugins/throttleconcurrents/ThrottleQueueTaskDispatcher.java

```diff
@@ -682,5 +689,26 @@ private int getMaxConcurrentPerNodeBasedOnMatchingLabels(
         return maxConcurrentPerNodeLabeledIfMatch;
     }
 
+    private void updatePauseAction(Task task, CauseOfBlockage cause) {
+        if (task instanceof PlaceholderTask) {
```

Comment:
`task` can also be of type `WorkflowJob`. Do we need to handle this as well?

Reply:
No, see above.